src/shared/components/Input/styles.ts

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+import Colors from 'shared/colors';
+import media from 'shared/media-types';
+
+type InputStyleProps = {
+  includeSendButton?: boolean;
+  validationState?: boolean;
+  validationMessage?: string;
+};
+
+export const InputRootContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-start;
+  width: 100%;
```

Comment:
```css
  display: flex;
  align-items: flex-start;
  justify-content: flex-start;
  width: 100%;
```
could be moved to a shared variable and used in InputRootContainer and InputContainer

Reply:
Done